keyboards/junco/keymaps/default/keymap.c

```diff
@@ -0,0 +1,162 @@
+        │SpdU│HueU│SatU│ValU│Rnxt│    │               │    │EClr│Rbt │DBUG│BOOT│    │
+        ├────┼────┼────┼────┼────┼────┤               ├────┼────┼────┼────┼────┼────┤
+        │SpdD│HueD│SatD│ValD│Rprv│RTgl│               │    │QWRT│COLE│    │    │    │
+        ├────┼────┼────┼────┼────┼────┼────┐     ┌────┼────┼────┼────┼────┼────┼────┤
+        │    │    │    │    │    │    │    │     │    │    │    │    │    │    │    │
+        └────┴────┴────┴────┴────┴────┴────┘     └────┴────┴────┴────┴────┴────┴────┘
+                       ┌───┬───┬───┬───┬───┐     ┌───┬───┬───┬───┬───┐
+                       │___│___│___│___│___│     │___│___│___│___│___│
+                       └───┴───┴───┴───┴───┘     └───┴───┴───┴───┴───┘
+    */
+    [_ADJUST] = LAYOUT_split4x6_r1(
+        KC_NO,     KC_NO,      KC_NO,      KC_NO,      KC_NO,      KC_NO,                       KC_NO,       KC_NO,         KC_NO,           KC_NO,   KC_NO,    KC_NO,
+        RGB_SPI,   RGB_HUI,    RGB_SAI,    RGB_VAI,    RGB_MOD,    KC_NO,                       KC_NO,       EE_CLR,        QK_RBT,          DB_TOGG, QK_BOOT,  KC_NO,
+        RGB_SPD,   RGB_HUD,    RGB_SAD,    RGB_VAD,    RGB_RMOD,   RGB_TOG,                     KC_NO,       DF(_QWERTY),   DF(_COLEMAK_DH), KC_NO,   KC_NO,    KC_NO,
+        KC_NO,     KC_NO,      KC_NO,      KC_NO,      KC_NO,      KC_NO,   KC_NO,    KC_NO,    KC_NO,       KC_NO,         KC_NO,           KC_NO,   KC_NO,    KC_NO,
+                               _______,    _______,    _______,    _______, _______,  _______,  _______,     _______,       _______,         _______
+    )
+    // clang-format on
+};
+
+/*
+    --- Rotary Encoder Mappings ---
+
+    Encoder mappings go from leftmost encoder to rightmost encoder on the physical board.
+    index 0 is the the optional leftmost encoder on the left half, index 1 is the right encoder
+    on the left half (by the thumb keys), index 2 is the left encoder on the right half (by the
+    thumb keys), and index 3 is the optional rightmost encoder on the right half.
+
+    If you are only using the 2 required encoders by the thumb keys, you only need to worry about
+    index 1 and index 2.
+
+    Note that the keycode for counter-clockwise rotation (CCW) goes first and then the key for
+    clockwise (CW) within ENCODER_CCW_CW.
+*/
+#ifdef ENCODER_MAP_ENABLE
+// clang-format off
+
+// Base layer encoder mappings:   index 0: mouse wheel up (CCW)/down (CW)     index 1: volume down/up           index 2: mouse wheel up/down                index 3: mouse wheel left/right
+#    define BASE_LAYER_ENCODERS { ENCODER_CCW_CW(KC_MS_WH_UP, KC_MS_WH_DOWN), ENCODER_CCW_CW(KC_VOLD, KC_VOLU), ENCODER_CCW_CW(KC_MS_WH_UP, KC_MS_WH_DOWN), ENCODER_CCW_CW(KC_WH_L, KC_WH_R) }
+const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][2] = {
+    // Base layers
+    [_QWERTY] =     BASE_LAYER_ENCODERS,
+    [_COLEMAK_DH] = BASE_LAYER_ENCODERS,
```

Comment:
just a heads up, this works fine (eg, will fall through to the qwerty layer).  I have a few keyboards configured this way (tractyl manuform, kyria, etc)
```suggestion
const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][2] = {
    // Base layers
    [_QWERTY] =     { ENCODER_CCW_CW(KC_MS_WH_UP, KC_MS_WH_DOWN), ENCODER_CCW_CW(KC_VOLD, KC_VOLU), ENCODER_CCW_CW(KC_MS_WH_UP, KC_MS_WH_DOWN), ENCODER_CCW_CW(KC_WH_L, KC_WH_R) },
    [_COLEMAK_DH] = { ENCODER_CCW_CW(KC_TRNS, KC_TRNS), ENCODER_CCW_CW(KC_TRNS, KC_TRNS), ENCODER_CCW_CW(KC_TRNS, KC_TRNS), ENCODER_CCW_CW(KC_TRNS, KC_TRNS) },
```